addons/web/static/src/js/chrome/keyboard_navigation_mixin.js

```diff
@@ -92,6 +103,9 @@ odoo.define('web.KeyboardNavigationMixin', function (require) {
          * @return {undefined|jQuery}
          */
         _hideAccessKeyOverlay: function () {
+            if (!this.$el) {
```

Comment:
not a big fan of this, i would prefer to bind the event handler when we're able to handle it.

Reply:
I'm not even sure why this is needed actually, some tests were breaking because $el was undefined, so that's why I added a destroy method, but it was still breaking. I would think that the web_client and the website_root instances shouldn't ever get destroyed so it surprised me. Maybe it's a problem with the tests' environment and I should fix the tests instead, I'll look into it.

Reply:
Well I guess you could set the KNavMixin on any element, not necessarily the root element, so removing the handlers in destroy is a good thing anyway. And for the same reason, I would register those handlers when the widget is started (and thus has a $el).

Reply:
Was gonna say the same thing as @aab-odoo